pkg/endpoint/endpoint_test.go

```diff
@@ -22,11 +22,11 @@ import (
 
 	"github.com/cilium/cilium/api/v1/models"
 	"github.com/cilium/cilium/common/addressing"
-	"github.com/cilium/cilium/pkg/comparator"
 	pkgLabels "github.com/cilium/cilium/pkg/labels"
 	"github.com/cilium/cilium/pkg/policy"
 	"github.com/cilium/cilium/pkg/policy/api"
 
+	"github.com/cilium/cilium/pkg/checker"
```

Comment:
Some of the format checkers may fail here. I think the convention is to import stdlib, cilium libs, then 3rd party ones. I noticed this in a few of the `*_test` files.

Thanks for the work!

Reply:
Thanks for catching this! Do you mean that the out-of-order `github.com/cilium/cilium/pkg/checker` import can cause some checkers to fail?

Reply:
yes! The order or the grouping of the imports. I forget the name but there is a script somewhere that runs go fmt and a few other tools and if the files change they fail the tests. In short, if you placed the `github.com/cilium/cilium/pkg/checker` import with the other cilium ones it would pass this check I'm describing.
I must be wrong, though, since the tests passed :/